packages/dhis2/src/Utils.js

```diff
@@ -14,9 +19,9 @@ export const CONTENT_TYPES = {
   xls: 'application/vnd.ms-excel',
 };
 
-export function buildUrl(urlString, hostUrl, apiVersion) {
+export function buildUrl(urlString, apiVersion) {
```

Comment:
We have `buildUrl` and we have `generateUrlPath`.

Do we _really_ need both? I'd love to consolidate these.

I think `buildUrl` is only called from inside `geneateUrlPath`, so we can probably just move the functionality inside?